jwst/regtest/test_fgs.py

```diff
@@ -0,0 +1,48 @@
+"""Regression tests for FGS Guidestar in ID and FINEGUIDE modes"""
```

Comment:
Minor nit, but the actual tests appear to be covering ID and ACQ1 modes, not FINEGUIDE.